### ISSUE TITLE
RayfireMesh Warnings

### ISSUE THREAD

**Issue by pbauman** [grinsfem/grins#441]
With GCC 6.1

<pre>
./../../../grinsfem/src/qoi/src/rayfire_mesh.C: In member function ‘void GRINS::RayfireMesh::coarsen(const libMesh::Elem*)’:
../../../../grinsfem/src/qoi/src/rayfire_mesh.C:537:44: warning: ‘end_node’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         elem->set_node(1) = _mesh->node_ptr(end_node->id());
                             ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
../../../../grinsfem/src/qoi/src/rayfire_mesh.C:536:44: warning: ‘start_node’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         elem->set_node(0) = _mesh->node_ptr(start_node->id());
                             ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
</pre>

@tradowsk please fix.

**Comment by tradowsk** [comment]
I shall get a PR out this afternoon. Weird the warnings don't show up on GCC5.3. Will test with GCC6.1 from now on

**Comment by tradowsk** [comment]
I just rebuilt the stack and compiled GRINS with GCC6.1, and I did not get a warning from the rayfire.

Would the `METHOD` affect that? I'm using `dbg` right now

**Comment by pbauman** [comment]
`METHOD=devel`

**Comment by pbauman** [comment]
Closed by #444
